real_intent/deliver/zapier/__init__.py

```
@@ -0,0 +1,110 @@
+from typing import Any
+from functools import partial
+from datetime import datetime
+from concurrent.futures import ThreadPoolExecutor
+
+from real_intent.internal_logging import log
+from real_intent.deliver.base import BaseOutputDeliverer
+from real_intent.schemas import MD5WithPII
+
+class ZapierDeliverer(BaseOutputDeliverer):
+
+    def __init__(self, webhook_urls: list[str], per_lead_insights: dict[str, str] = {}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)